bindings/python/tests/bindings/test_pre_tokenizers.py

```diff
@@ -132,6 +132,7 @@ def test_can_modify(self):
 class TestPunctuation:
     def test_instantiate(self):
         assert Punctuation() is not None
+        assert Punctiation("removed") is not None
```

Comment:
```suggestion
        assert Punctuation("removed") is not None
```

Reply:
Not sure how it slipped in as I tried to run the tests locally (probably not the right way?).